src/components/ArticleCards.jsx

```diff
@@ -0,0 +1,32 @@
+import Button from 'react-bootstrap/Button';
+import Card from 'react-bootstrap/Card';
+import { Link } from 'react-router-dom';
+
+import { getArticles } from '../utils/api';
+
+export default function ArticleCards (props) {
+    const {article, setArticles} = props
+
+    function handleClick(event) {
+      console.log("submitted")
+      getArticles().then((articlesData)=>{
+        articlesData.map((article) => {
```

Comment:
unsure what this map is supposed to be doing, please make sure you complete code before pushing it up! :)